src/components/UserSettingsModal/UserSettingsModa.service.ts

```diff
@@ -0,0 +1,5 @@
+import { env } from './../../env/environment';
+import { Fetch } from './../../helpers/fetch';
+export function getUserData(id: number): Promise<any> {
+  return Fetch.request(env.securedRoutes + '/user/' + id);
```

Comment:
use template `${....}`

Reply:
File name should be `UserSettingsModal.service.ts`.